yellowbrick/model_selection/CVScores.py

```diff
@@ -0,0 +1,174 @@
+    
+    Notes
+    -----
+    
+    This visualizer is a wrapper around for the ``sklearn.model_selection.cross_val_score``
+    <<http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.cross_val_score.html>>
+    
+    Refer to the scikit-learn
+    `cross-validation guide <http://scikit-learn.org/stable/modules/cross_validation.html>`
+    for more details
+    
+    """
+
+    def __init__(self, model, ax=None, cv=None, scoring=None, **kwargs):    
+                
+        super(CVScores, self).__init__(model, ax=ax, **kwargs)
+    
+        self.cv = cv
+        self.scoring = scoring
+
+    def fit(self, X, y, **kwargs):
+        """
+        Fits the learning curve with the wrapped model to the specified data.
+        Draws training and test score curves and saves the scores to the
+        estimator.
+
+        Parameters
+        ----------
+        X : array-like, shape (n_samples, n_features)
+                Training vector, where n_samples is the number of samples and
+                n_features is the number of features.
+
+        y : array-like, shape (n_samples) or (n_samples, n_features), optional
+                Target relative to X for classification or regression;
+                None for unsupervised learning.
+
+        Returns
+        -------
+        self : instance
+        
+        """
+
+        self.cv_scores_ = cross_val_score(self.estimator, X, y, cv=self.cv, scoring=self.scoring)
+        self.cv_scores_mean_ = np.mean(self.cv_scores_)
+
+        self.draw()
+        return self
+
+    def draw(self, **kwargs):
+        """
+        creates the bar chart of the CV scores generated from the fit method and places 
+        a dashed horizontal line that represents the average value of the CV scores
+        """
+        minimum = min(self.cv_scores_)
+        if minimum > 0.05:
+            minimum = minimum - 0.05 
+        plt.ylim(minimum, 1)
+        xvals = range(1, len(self.cv_scores_) + 1, 1)
+        width = kwargs.pop("width", 0.3)
+        self.ax.bar(xvals, self.cv_scores_, width = width)
+        color = kwargs.pop("color", "b")
+        linewidth = kwargs.pop("linewidth", 1)
+        self.ax.axhline(self.cv_scores_mean_, color=color, label='Average', linestyle='--', linewidth=linewidth)
+
+        return self.ax
+
+    def finalize(self, **kwargs):
+        """
+        Add the title, legend, and other visual final touches to the plot.
+        """
+        # Set the title of the figure
+        self.set_title('Cross Validation Scores for {}'.format(self.name))
+
+        # Add the legend
+        loc = kwargs.pop("loc", "best")
+        edgecolor = kwargs.pop("edgecolor", "k")
+        self.ax.legend(frameon=True, loc=loc, edgecolor=edgecolor)
+
+        #set spacing between the x ticks
+        self.ax.xaxis.set_major_locator(ticker.MultipleLocator(1))
+
+        # Set the axis labels
+        self.ax.set_xlabel('Training Instances')
+        self.ax.set_ylabel('Score')
+
+
```

Comment:
nice work on the visualizer.  I caught a couple of things.  Let me know if you need me to clarify any of this.

1.  CVScores.py is missing a "quick method".  Look at the method found at end of the validationcurve.py code for an example of what I mean

Reply:
2.  The CVScore method isn't callable.  Meaning I tried to run the example with your code and I get an error.  The solution is to edit the __init__.py file that is in the same directory as CVScores.py.  You need to add an import statement similar to the ones already present in __init__.py